x-pack/plugins/enterprise_search/server/routes/enterprise_search/engines.test.ts

```diff
@@ -64,72 +83,23 @@ describe('engines routes', () => {
     });
   });
 
-  describe('POST /internal/enterprise_search/engines', () => {
```

Comment:
now that we are removing the post, do we have a ticket to address the frontend change to pass the create flag? I would also suggest to do the frontend change in this PR as merging will break the experience

Reply:
> now that we are removing the post, do we have a ticket to address the frontend change to use the PUT and add the create flag?

I think we don't have one yet. cc: @kderusso 

> I would also suggest to do the frontend change in this PR as merging will break the experience

Yeah, we initially thought of having the changes in separate [PR](https://github.com/elastic/kibana/pull/153305). But if it make sense too, Will add those changes in here.